### PR TITLE
#16: Resolve the variables on the fly

### PR DESCRIPTION
**Pull request by essobedo**
A fix proposal for https://github.com/mucommander/mucommander/issues/16:

This PR allows to resolve the variables of type _$[var-name]_ at initialization time thanks to a decorator.

### CODE REVIEW

File: src/main/java/com/mucommander/text/Translator.java [hunk 1]
```diff
@@ -222,36 +220,14 @@ public static String get(String key, String... paramValues) {
...
-                                buffer.append(value, matcher.start(), matcher.end());
-                            }
-                            startIndex = matcher.end();
-                        }
-                        buffer.append(value.substring(startIndex));
-                        result = buffer.toString();
-                        cache.putIfAbsent(key, (String) result);
-                    }
-                }
+                return resourceBundle.getObject(key);
```
Comment (ahadas) [inline src/main/java/com/mucommander/text/Translator.java]: so do we still need the code in lines 229-231 and not just to return whatever is stored in the map?

**Comment by ahadas** [comment]
very nice contribution! probably an issue that caused by the move to properties files.. minor comment inside. sorry for the long time it took me to get to it. please address the comment inside and lets merge it

File: src/main/java/com/mucommander/text/Translator.java (hunk 1 above)
Comment (essobedo) [inline src/main/java/com/mucommander/text/Translator.java]: @ahadas Actually we still need it as it only keep key/value pairs whose value contains at least one variable as you can see [here](https://github.com/mucommander/mucommander/pull/17/files#diff-1517cfcc43a86b331edf0ab843b6b0b1R280)

Comment (essobedo) [inline src/main/java/com/mucommander/text/Translator.java]: @ahadas Actually we still need it as it only keep key/value pairs whose value contains at least one variable as you can see [here](https://github.com/mucommander/mucommander/pull/17/files#diff-1517cfcc43a86b331edf0ab843b6b0b1R280)

**Comment by ahadas** [comment]
Thank you for this contribution.

**Pull request merged**